crates/ide_assists/src/handlers/extract_function.rs

```diff
@@ -63,16 +64,11 @@ pub(crate) fn extract_function(acc: &mut Assists, ctx: &AssistContext) -> Option
         return None;
     }
 
-    let node = ctx.covering_element();
-    if node.kind() == COMMENT {
-        cov_mark::hit!(extract_function_in_comment_is_not_applicable);
-        return None;
-    }
-
-    let node = match node {
```

Comment:
I think we can keep this branch, this would only be hit if all the user did was select a comment, in which case showing the assist is pointless